apps/js_urls/templatetags/js_urls.py

```diff
@@ -0,0 +1,22 @@
+# Django Imports
+from django import template
+from django.utils.safestring import mark_safe
+
+# HTK Imports
+from htk.apps.js_urls.serializer import get_urls_as_json
```

Comment:
Too many functions name `get_*`

Reply:
This function renamed to `build_routes` as you suggested.